insights/parsers/rdma_config.py

```diff
@@ -2,13 +2,14 @@
 RdmaConfig - file ``/etc/rdma/rdma.conf``
 =========================================
 """
-from .. import get_active_lines, Parser, parser, LegacyItemAccess
+from .. import get_active_lines, Parser, parser
 from insights.parsers import split_kv_pairs
 from insights.specs import Specs
+from insights.parsers import SkipException
 
 
 @parser(Specs.rdma_conf)
-class RdmaConfig(Parser, LegacyItemAccess):
+class RdmaConfig(Parser):
```

Comment:
Just noticed that you removed the `LegacyItemAccess`. It's not good to do so.

Reply:
updated the changes